bg3-modders-multitool/bg3-modders-multitool/Services/PathHelper.cs

```diff
@@ -0,0 +1,65 @@
+﻿using bg3_modders_multitool.Properties;
+using System;
+using System.IO;
+
+namespace bg3_modders_multitool.Services
+{
+    public class PathHelper
+    {
+        private static readonly PathHelper _instance = new PathHelper();
+        public static PathHelper Instance => _instance;
+
+        public static string ModsFolderPath { get; private set; }
+        public static string PlayerProfilesFolderPath { get; private set; }
+
+        private PathHelper()
+        {
+            InitializePaths();
+        }
+
+        private void InitializePaths()
+        {
+            try
+            {
+                // Gets the current config value for the game path..
+                var gameDocumentsPath = Settings.Default.gameDocumentsPath;
+
+                if (string.IsNullOrEmpty(gameDocumentsPath))
+                {
+                    // As the config value is empty, we'll try to find the game path automatically by
+                    // looking for the default location..
+                    gameDocumentsPath = Path.Combine(
+                        Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData, Environment.SpecialFolderOption.DoNotVerify),
+                        "Larian Studios",
+                        "Baldur's Gate 3"
+                    );
+                }
+
+                // In case the game path is still empty, we'll throw an exception...
+                if (!Directory.Exists(gameDocumentsPath))
+                    throw new Exception($"Game Documents path not found at '{gameDocumentsPath}'.");
```

Comment:
Do not use try catch in this way, just print to console. However, at this point during initialization, the console window context does not exist, so you can't do this.

Reply:
Got it. I've restructured the way the PathHelper manages folder path initialization. Now, instead of relying on an exception and subsequently on the WriteToConsole() method, which leads to a null exception error, both paths will remain empty in case of an error. The MainWindow view will utilize a model property to notify the user of a misconfiguration that requires attention:

1. A console message will explicitly inform the user that the mods/profiles path could not be located.
2. The "needConfig" label will be displayed, similar to when the BG3 or Divinity paths are missing.